Sources/ScipioKit/Producer/PIF/FrameworkComponentsCollector.swift

```diff
@@ -4,7 +4,12 @@ import PackageModel
 
 /// FileLists to assemble a framework bundle
 struct FrameworkComponents {
+    /// Whether the built framework is a versioned bundle or not.
+    ///
+    /// ref: https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPFrameworks/Concepts/FrameworkAnatomy.html
```

Comment:
It's better to use DocC format

```suggestion
    /// - seealso: https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPFrameworks/Concepts/FrameworkAnatomy.html
```